src/sections/Community/Handbook/leadership.js

```diff
@@ -0,0 +1,139 @@
+            <h2>What does leadership mean to the Layer5 community?</h2>
+            <p>
+              A leader is someone who can contribute to the Layer5 Community's growth by being accountable, participating in decision-making, and feeling responsible.
+            </p>
+            <h3>What does it take to be a leader?</h3>
+            
+            <h4><strong>Community Manager</strong></h4>
+            <p>
+              Roles/Responsibilities:
+              <ul>
+                <li>Moderating, engaging, and supporting community members on platforms like Slack, GitHub etc.</li>
+                <li>Building healthy relationships among community members.</li>
+                <li>Celebrating community successes, sending swag, and recognizing top contributors.</li>
+                <li>Regularly updating the community on the metrics performance.</li>
+                <li>Creating and managing new community programs.</li>
+                <li>Organizing meetups, events, and other engagements.</li>
+                <li>Coordinating with other departments—such as product, engineering, and content marketing—to support community initiatives.</li>
+              </ul>
+            </p>
+            <p><strong>Checklist before becoming a Community Manager</strong></p>
+            <input type="checkbox" name="CM-checklist-1" id="CM-checklist-1"/>
+            <label htmlFor="CM-checklist-1">
+              Prior technical community management experience
+            </label>
+            <input type="checkbox" name="CM-checklist-2" id="CM-checklist-2"/>
+            <label htmlFor="CM-checklist-2">
+              Self-motivated with the ability to manage multiple competing priorities
+            </label>
+            <input type="checkbox" name="CM-checklist-3" id="CM-checklist-3"/>
+            <label htmlFor="CM-checklist-3">
+              A growth mindset and an approach to new information and knowledge with curiosity
+            </label>
+            <input type="checkbox" name="CM-checklist-4" id="CM-checklist-4"/>
+            <label htmlFor="CM-checklist-4">
+              A challenge seeker who desires and readily takes on new challenges and works towards solutions
+            </label>
+            <input type="checkbox" name="CM-checklist-5" id="CM-checklist-5"/>
+            <label htmlFor="CM-checklist-5">
+              Good communication skills
+            </label>
+            <input type="checkbox" name="CM-checklist-6" id="CM-checklist-6"/>
+            <label htmlFor="CM-checklist-6">
+              Strong relational skills
+            </label>
+            <input type="checkbox" name="CM-checklist-7" id="CM-checklist-7"/>
+            <label htmlFor="CM-checklist-7">
+              An understanding of how contemporary open-source projects function
+            </label>
+            <br/>
+            <h4><strong>Next you be a <Link to="/community/meshmates">MeshMate</Link></strong></h4>
```

Comment:
```suggestion
            <h4><strong> <Link to="/community/meshmates">MeshMate</Link></strong></h4>
```
Let's just keep it as MeshMate.